src/controllers/mainController.ts

```diff
@@ -408,7 +411,7 @@ export default class MainController implements vscode.Disposable {
     /**
      * Verifies the extension is initilized and if not shows an error message
      */
-    private CanRunCommand(): boolean {
+    private canRunCommand(): boolean {
```

Comment:
Mine is checked in at this point so you'll need to merge and fix the conflicts unfortunately.